### DISCUSSION TITLE
New Safari (Version 17.0) can't open GitHub Pages Drop-Down "Build and deployment  > Source"

### DISCUSSION THREAD

**Discussion by michaelweinold** [community/community#68580] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

The drop-down under `Settings > Pages > Build and deployment > Source` cannot be opened in Safari 17.0 (no content blockers):

https://github.com/community/community/assets/23102087/6cc5b205-2ed8-4eea-981c-3e57bfdbe4c5

This works fine in Google Chrome.

**Comment by pure-engineering** [comment]
To be honest, GitHub has a lot of bugs in Safari 17.4 - almost 20% of the features don't work correctly including PR reviews
